.rubocop.yml

```diff
@@ -4,3 +4,7 @@ AllCops:
   Exclude:
   - 'db/schema.rb'
   - 'vendor/**/*'
+
+Style/Documentation:
```

Comment:
I'd want to highlight this in the "Why" of the commit, in case somebody in the future wanted to understand the reasoning behind excluding this.

Reply:
Thanks Adrian. Comments added in response to both points.